src/jit/flowgraph.cpp

```diff
@@ -17776,6 +17776,10 @@ BasicBlock* Compiler::fgAddCodeRef(BasicBlock* srcBlk, unsigned refData, Special
 #if defined(UNIX_X86_ABI)
         codeGen->setFrameRequired(true);
         codeGen->setFramePointerRequiredGCInfo(true);
+        add->acdStkLvl = 0;
+#ifdef DEBUG
+        add->acdDstBlk->bbTgtStkDepth = 0;
+#endif
 #else  // !defined(UNIX_X86_ABI)
```

Comment:
Are these changes necessary? It doesn't look like `CodeGen::genAdjustStackLevel()` requires them.